chapters/stack/functions/drills/tasks/string_print/solution/print_string.asm

```diff
@@ -0,0 +1,23 @@
+%include "../utils/printf32.asm"
+
+section .data
+    mystring db "This is my string", 0
+
+section .text
+
+extern puts
+extern printf
+global main
+main:
+    push ebp
+    mov ebp, esp
+
+    PRINTF32 `[PRINTF32]: %s\n[PUTS]: \x0`, mystring
+
+
+    push mystring
+    call puts
+    add esp, 4
```

Comment:
Students will not understand why this `add esp, 4` is needed when the call returns